airbyte-integrations/connectors/destination-google-sheets/destination_google_sheets/buffer.py

```diff
@@ -0,0 +1,118 @@
+#
+# Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+#
+
+
+from typing import Any, List, Mapping
+
+from airbyte_cdk import AirbyteLogger
+from airbyte_cdk.models import AirbyteStream
+
+
+class WriteBuffer:
+
+    # Default instance of AirbyteLogger
+    logger = AirbyteLogger()
+    # Buffer for input records
+    records_buffer = {}
+    # Placeholder for streams metadata
+    stream_info = {}
+    # interval after which the records_buffer should be cleaned up for selected stream
+    flush_interval = 1000
+
+    @property
+    def default_missing(self) -> str:
+        """
+        Default value for missing keys in record stream, compared to configured_stream catalog.
+        Overwrite if needed.
+        """
+        return ""
+
+    def buffer_stream(self, configured_stream: AirbyteStream):
+        """
+        Saves important stream's information for later use.
+
+        Particulary, creates the data structure for `records_stream`.
+        Populates `stream_info` placeholder with stream metadata information.
+        """
+        stream = configured_stream.stream
+        self.records_buffer[stream.name] = []
+        self.stream_info[stream.name] = {"headers": sorted(list(stream.json_schema.get("properties").keys())), "is_set": False}
```

Comment:
list is not needed here

Reply:
It's needed, because we need a placeholder for the future values.